_includes/templates/install/cassandra-rhel-install.md

```diff
@@ -1,5 +1,45 @@
-Please follow [official Apache Cassandra installation guide](https://cassandra.apache.org/doc/latest/cassandra/installing/installing.html#install-as-rpm-package).
+Create and edit the Cassandra repository file:
```

Comment:
I prefer a single copy-paste command. It is much clear and usable from my point of view. 

```bash
sudo tee /etc/yum.repos.d/cassandra.repo > /dev/null <<EOF
Line 1
Line 2
Line 3
EOF
```